keyboards/xdboards/recon/keyboard.json

```diff
@@ -0,0 +1,215 @@
+{
+    "manufacturer": "XD Boards",
+    "keyboard_name": "xdboards/recon",
+    "maintainer": "Xanimos",
+    "development_board": "promicro",
+    "pin_compatible": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "oled": true
+    },
+    "matrix_pins": {
+        "cols": ["D4", "C6", "D7", "E6", "B4", "B5"],
+        "rows": ["F4", "F5", "F6", "F7", "B1"]
+    },
+    "url": "https://xdboards.xyz",
+    "tags": ["columnar", "split", "thumb cluster", "rgb", "encoder"],
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x02F7",
+        "vid": "0xFEED"
+    },
+    "encoder": {
+        "rotary": [
+            { "pin_a": "B6","pin_b": "B2" }
+        ]
+    },
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "B3",
```

Comment:
Actually, this won't work on AVR, which is what this keyboard targets by default. B3 is not a UART capable pin.

Reply:
I see that, trying to find a solution. Have some time to help on discord?

Reply:
Sure, feel free to create a forum post in the QMK discord (help-forum).

Reply:
I reworked the board to use the serial pins allowed by AVR controllers. I only used RP2040 controllers while building.